packages/strapi/lib/middlewares/public/index.js

```diff
@@ -116,7 +92,7 @@ module.exports = strapi => {
           await next();
         },
         koaStatic(buildDir, {
-          index: 'index.html',
+          index: '/index.html',
```

Comment:
index isn't the path of the index but the name of the index file :) the `/` isn't required

Reply:
Damn it